### DISCUSSION TITLE
Make section start position MIDI-controllable

### DISCUSSION THREAD

**Discussion by Acousmachin** [helgoboss/helgobox#978] [Ideas]
I whish that we could dynamically change the start position of clips with a MIDI controller or velocity...
Percentage position would be nice for me (non beat clips).

**Comment by helgoboss** [comment]
I'm going to make multiple things MIDI-controllable in a row. This discussion will be the main one. For the sake of completeness, I refer to other similar discussions and close them: #1262, #1266
